ssh/ssh.go

```diff
@@ -388,3 +389,20 @@ func (ctx *SSHContext) SetPermissions(host Host, path string, permissions string
 
 	return nil
 }
+
+func (ctx *SSHContext) WaitForMountPoints(host Host, path string) (err error) {
+	cmd, err := ctx.SudoCmd(host, "systemd-run", "--wait", "--property=RequiresMountsFor="+path, "true")
```

Comment:
Nit: use the absolut path to `/run/current-system/sw/bin/systemd-run`  just in case the default of the system has been changed.

Reply:
Thanks, updated.